src/tests/decider-distro-tests-rs/tests/utils/control.rs

```diff
@@ -0,0 +1,149 @@
+use crate::utils;
+use crate::utils::spell;
+use connected_client::ConnectedClient;
+use eyre::WrapErr;
+use fluence_spell_dtos::trigger_config::TriggerConfig;
+use fluence_spell_dtos::value::ScriptValue;
+use maplit::hashmap;
+use serde::Deserialize;
+use serde_json::{json, Value};
+use std::fs;
+use std::path::PathBuf;
+use std::time::Duration;
+
+pub async fn update_config(
+    client: &mut ConnectedClient,
+    trigger_config: &TriggerConfig,
+) -> eyre::Result<Vec<Value>> {
+    utils::execute(
+        client,
+        r#"(call relay ("spell" "update_trigger_config") ["decider" config])"#,
+        "\"done\"",
+        hashmap! {
+            "config" => json!(trigger_config),
+        },
+    )
+    .await
+}
+
+// God left me here
+pub fn modify_decider_spell_script(
+    tmp_dir: PathBuf,
+    decider_spell_id: String,
+    updated_script: String,
+) {
+    let script_path: PathBuf = tmp_dir.join(
+        [
+            "services",
+            "workdir",
+            &decider_spell_id,
+            "tmp",
+            "script.air",
+        ]
+        .iter()
+        .collect::<PathBuf>(),
+    );
+
+    fs::write(&script_path, updated_script).unwrap();
```

Comment:
Why don't you do this via service calls?

Reply:
It's not possible to do in the tests in a nice way. 

I researched the question long ago, afaik we even discussed it on the daily several times, and none of the suggestions worked, so I did this. We could provide some API in `created_swarm,` but it wasn't a priority since this solution worked out, though it's rough.